activerecord-typedstore.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ['lib']
 
-  spec.add_dependency 'activerecord', '>= 3.2', '< 5'
+  spec.add_dependency 'activerecord', '>= 3.2', '< 5.1'
```

Comment:
Should we `>= 4.2` as well? I'm totally in favor of bumping to `1.0` and drop `< 4.2` support so we can focus on the AR attributes API